modules/doc/content/newsletter/2023/2023_02.md

```diff
@@ -109,11 +110,32 @@ at a temperature given by a post-processor value.
 
 We have improved the recently-added capability that allows a user to [mark a solution as "invalid"](source/problems/SolutionInvalidity.md). Once marked invalid, the system allows output of which object triggered the invalidity as well as how many times the warning occurs. An invalid solution means that the solution somehow does not satisfy requirements, such as a value being out of the bounds of a correlation.  Solutions are allowed to be invalid *during* the nonlinear solve - but are not allowed to be invalid once it converges. A "converged" solution that is marked as invalid will cause MOOSE to behave as if the solution did NOT converge - including cutting back timesteps, etc. A macro, `flagInvalidSolution`, is provided to declare a solution as "invalid". The macro allows for a user-defined short description of the object (optional) and a message to describe the invalidity.
 
-## libMesh-level Changes
+### Performance Improvements for Navier-Stokes Finite Volume Simulations
+
+The automatic porosity jump handling feature introduced in the last newsletter
+slowed down Navier-Stokes finite volume simulations. A representative example
+was the mixing length channel test in the incompressible finite volume test
+suite, which went from a run-time of ~100 seconds to ~121 seconds on a
+twice-refined mesh. To combat this slowing, we introduced several optimizations
+including:
+
+- Building a boundary ID to Dirichlet boundary condition map during the
+  simulation's initial setup as opposed to querying for boundary conditions
+  every time a new face is visited
+- Disabling pre-initialized data computation when only the on-the-fly functor
+  system is used
+- Caching the degree of freedom indices and re-using them when the same element
+  is repeatedly visited
 
-## PETSc-level Changes
+With these changes, the same mixing length test (twice refined) reduced its
+run-time to ~83 seconds, a 20% improvement over the pre-porosity-jump treatment behavior,
+and a 45% improvement over the post-porosity-jump treatment behavior.
 
 ## Bug Fixes and Minor Enhancements
 
 - `SimpleFluidProperties` has been further developed to better support the (specific volume, specific energy)
   variable set used in the thermal_hydraulics module.
+- The HFEM flavor using Lagrange multipliers on lower-dimensional elements now
+  supports distributed meshes.
+- The [Bounds system](Bounds/index.md) was extended from supporting only first order Lagrange variables to also
+  supporting constant monomial variables. This system currently does not include support for finite volume variables.
```

Comment:
```suggestion
  supporting constant monomial variables, including support for finite volume variables.
```

I just fixed that :), before the 28th at midnight

Reply:
😆 Sounds good!